docs/blog/2018-07-07-graphic-design-class/index.md

```diff
@@ -21,9 +21,9 @@ However, in his current class _nobody has dropped out_. And every single one of
 
 ## Here are some of the student’s final sites:
 
-[![Image of graphic designer's site](graphic-design-1.png)](http://mmd-site.netlify.com/)
+[![Image of graphic designer's site](graphic-design-1.png)](http://mywetpaintstudio.com/)
 
-[Melany Dierks](http://mmd-site.netlify.com/) ([source](https://github.com/reguv760/mmd-site))
+[Melany Dierks](http://mywetpaintstudio.com/) ([source](https://github.com/reguv760/mmd-site2))
```

Comment:
this website is not loading to me. can someone else doublecheck?
cc @Darking360 @nucliweb